Resources/Prototypes/Entities/Mobs/NPCs/xeno.yml

```diff
@@ -431,3 +466,234 @@
         Base: dead_small_purple_snake
   - type: SolutionTransfer
     maxTransferAmount: 1
+
+# NPC versions
```

Comment:
NPC versions should be the parent and non-NPC should be the child

Reply:
That is how it is, there's just the one parent that they all inherit from...but I see I need a non-npc version of it.

Reply:
OK...I'll reconfigure it that way